### DISCUSSION TITLE
Question which Python model to use 3.10 or anything above will do?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#1815] [Q&A] [answered]
that's it, that's my question

**Comment by brknsoul** [comment, marked as answer]
Known working versions are 3.10.6 and 3.10.11, but any 3.10.x version should work.

3.11, however, does not work (yet?).

For Windows, you can nab 3.10.11 [here](https://www.python.org/downloads/release/python-31011/), or 3.10.6 [here](https://www.python.org/downloads/release/python-3106/), downloads are found near the bottom of the page; "Windows installer (64-bit)"

**Reply by weirdlighthouse** [reply]
thanks
